help/en/html/FAQ.shtml

```diff
@@ -87,115 +87,115 @@
-        <a href="apps/PanelPro/GettingStarted.shtml">tutorial</a>).  After that, you can go won to operations via DispatcherPro and OperationsPro 
-        or other forms of automation - use whatever features suit your needs.
+        <dd>Many people start by connecting their DCC system controller and registering their
+        locomotives using <a href="apps/DecoderPro/index.shtml">DecoderPro </a>.  You can then start running trains using a throttle 
+        in JMRI, a <a href="../package/jmri/jmrit/withrottle/UserInterface.shtml">wireless throttle, or your smartphone / tablet.</a>  Alternatively,
+        you can dig right in entering layout information on a Layout Editor Panel in PanelPro (<a href="apps/PanelPro/GettingStarted.shtml">tutorial</a>).
+        Before moving on to DispatcherPro and OperationsPro, make sure you have a working Panel. More advanced operations using these modules requires a 
+        more complex setup before they will work properly. The KEY... Don't do too much at once.  Get the more basic aspects of JMRI working before you take
+        the next step.
         </dd>
 
-        <dt id="wherelearn">Where can I learn about using JMRI?</dt>
+        <dt id="wherelearn"><strong>Where can I learn about using JMRI?</strong></dt>
 
         <dd>
-          There are many presentations and videos available on the internet to help in learning the different 
-          aspects of JMRI.  In addition, the JMRI.org web site has getting started guides (PanelPro, <a href="apps/DecoderPro/Tour.shtml">DeocderPro</a>).
-          Reading questions asked by others, or asking your own, on the user forum can also be a way to learn.  
+          The internet - There are many presentations and videos available to help in learning different 
+          aspects of JMRI.  The JMRI.org web site has getting started guides (<a href="apps/PanelPro/GettingStarted.shtml">PanelPro</a>, <a href="apps/DecoderPro/Tour.shtml">DecoderPro</a>).
+          Reading questions and responses asked by others on the user forum. Asking your own questions on the forum.  
         </dd>
 
-        <dt id="apps">What is the difference between using PanelPro, DecoderPro, and the other JMRI applications?</dt>
+        <dt id="apps"><strong>What is the difference between using PanelPro, DecoderPro, and the other JMRI modules?</strong></dt>
 
         <dd>
-         JMRI can be viewed as a series of functions and user interfaces operating on a variety of data - all linked together.  
-         DecoderPro, PanelPro,<a href="apps/DispatcherPro/index.shtml"> DispatcherPro, </a>
+         The JMRI application is a series of functions and user interfaces operating on a common linked data structure.  
+         While <a href="apps/DecoderPro/index.shtml">DecoderPro </a>, <a href="apps/PanelPro/index.shtml">PanelPro </a>,<a href="apps/DispatcherPro/index.shtml"> DispatcherPro, </a>
```

Comment:
Line 141: we try to avoid using too techy languages, like "common linked data structure".

Reply:
Changed